server-master/src/main/scala/com/aliyun/emr/rss/service/deploy/master/Master.scala

```diff
@@ -670,41 +682,23 @@ private[deploy] class Master(
       }
     isActive
   }
+
+  override def initial(): Unit = {
```

Comment:
I think init() or initialize() is better.

Reply:
Done